pkg/types/service_offering.go

```diff
@@ -46,6 +46,8 @@ type ServiceOffering struct {
 	CatalogName string `json:"catalog_name"`
 
 	Plans []*ServicePlan `json:"plans"`
+
+	LastOperation *Operation `json:"last_operation,omitempty"`
```

Comment:
Why do we need last operation for offerings and plans?

Reply:
Not really sure, which entity support async, but I will remove this